aea/agent.py

```diff
@@ -71,7 +71,7 @@ def __init__(self, name: str,
         :return: None
         """
         self._name = name
-        self._crypto = Crypto(private_key_pem_path=private_key_pem_path)
+        self._crypto = Wallet(private_key_pem_path=private_key_pem_path)
```

Comment:
I am tempted to use `wallet` throughout in place of `crypto`. Any thoughts on this @marcofavorito ?